docs/spec/overview.md

```diff
@@ -31,7 +31,7 @@ For more details, see [Interface Contracts](interfaces.md).
 
 <!-- This image is sourced from https://drive.google.com/open?id=1o_0Xh5VjwpQ7Px08h_Q4qnaOdMjt4yCEPixRFwJQjh8 -->
 
-<img alt="Resource Types ClusterChannelProvisioners" src="images/resource-types-provisioner.svg" width="200">
+![Resource Types ClusterChannelProvisioners](images/resource-types-provisioner.png)
```

Comment:
@Harwayne @vaikas-google I add to add a PNG ... (downloaded from the referenced Google doc)

the new rules hate inline HTML .. which is OK, but the suggestions about sizes seem not to work on GH ...